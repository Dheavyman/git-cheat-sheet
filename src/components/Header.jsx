import React from 'react';

import SearchBar from './SearchBar';

/**
 * Header component
 *
 * @returns {object} React element
 */
const Header = () => (
  <div className="row header">
    <div className="col s1 push-s11">
      <a href="#!" data-target="slide-out" className="sidenav-trigger">
        <i className="material-icons">menu</i>
      </a>
    </div>
    <div className="col s11 pull-s1">
      <SearchBar />
    </div>
  </div>
);

export default Header;
