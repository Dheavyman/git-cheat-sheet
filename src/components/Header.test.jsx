import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';


import Header from './Header';

describe('Header component', () => {
  it('should render component successfully', () => {
    const renderer = new ShallowRenderer()
    renderer.render(<Header />);
    const result = renderer.getRenderOutput()
    expect(result).toMatchSnapshot();
  });
});
