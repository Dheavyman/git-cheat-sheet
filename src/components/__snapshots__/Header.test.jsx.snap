// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render component successfully 1`] = `
<div
  className="row header"
>
  <div
    className="col s1 push-s11"
  >
    <a
      className="sidenav-trigger"
      data-target="slide-out"
      href="#!"
    >
      <i
        className="material-icons"
      >
        menu
      </i>
    </a>
  </div>
  <div
    className="col s11 pull-s1"
  >
    <Connect(SearchBar) />
  </div>
</div>
`;
